treeherder/perf/management/commands/synthesize_backfill_report.py

```diff
@@ -41,17 +38,9 @@ def add_arguments(self, parser):
         )
 
     def handle(self, *args, **options):
-        frameworks, repositories, max_timestamp, days_number = self._parse_args(**options)
+        frameworks, repositories, start_time, days_interval = self._parse_args(**options)
         self._validate_args(frameworks, repositories)
-
-        summaries_to_retrigger = self._fetch_by(
-            self._summaries_since(max_timestamp),
-            frameworks,
-            repositories
-        )
-        json_output = self._identify_retriggerables(summaries_to_retrigger, days_number)
-
-        print(json_output)
+        print(IdentifyRetriggerableAlerts()(frameworks, repositories, start_time, days_interval))
```

Comment:
We should rename the `days_interval` to something more intuitive. Maybe `data_points_lookup_interval` & provide extra details in the doc strings?

Reply:
Ok. Agree with this, it will be changed inside class definition. But I suggest using `days_interval` for instantiation as is useful to be explicit about the interval used .